scripts/summit-cli.mjs

```
@@ -0,0 +1,861 @@
+      break;
+
+    case 'reset':
+      log.warn('This will DELETE ALL DATA. Press Ctrl+C to cancel.');
+      await new Promise(resolve => setTimeout(resolve, 3000));
+      exec('docker-compose -f docker-compose.dev.yml down -v');
+      log.success('Databases reset');
+      break;
+
+    case 'shell':
+      const dbType = args._[2] || 'postgres';
+      if (dbType === 'postgres') {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)